### DISCUSSION TITLE
Broken active marker deck

### DISCUSSION THREAD

**Discussion by NageshEranki** [bitcraze/discussions#1494] [Q&A]
Hi 

We have a few active marker decks in the lab here. I've identified a couple that don't work properly. Are there any troubleshooting steps one can follow to understand what's going on? Since these decks are quite expensive, we would really prefer not having to buy new ones. 

Thanks

**Comment by knmcguire** [comment]
hi!

First thing I'd look at is the console tab when connecting to a crazyflie with one of these attached. Could you share the output of that?
